lenovo/yoga/6/13ALC6/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, pkgs, ...  }:
+
+{
+  imports = [
+    ../../../thinkpad/yoga.nix
+    ../../../../common/gpu/amd/default.nix
+  ];
+
+  boot.initrd.kernelModules = [ "ideapad_laptop" ];
+  hardware.opengl.extraPackages = with pkgs; [
+    vaapiVdpau
+    libvdpau-va-gl
+  ];
+
+  # latest kernel needed to make wifi work
+  boot.kernelPackages =  lib.mkIf (lib.versionOlder pkgs.linux.version "5.16") pkgs.linuxPackages_latest;
+
+  # energy savings
+  hardware.bluetooth.powerOnBoot = lib.mkDefault false;
```

Comment:
I would also leave this for your own configuration.nix. Some people might use bluetooth keyboards for logging in...

```suggestion
```